pkg/config/config.go

```diff
@@ -52,15 +52,18 @@ type Interface interface {
 
 type Dumper struct {
 	Configs  []v1beta1.Config
+	name     string
 	RootPath string
 }
 
-func NewConfiguration(rootPath string, configs []v1beta1.Config) Interface {
+func NewConfiguration(name, rootPath string, configs []v1beta1.Config) Interface {
```

Comment:
convert this exported function to builder mode like using `OptionFunc` to avoid modification frequently?